projects/aca-content/src/lib/components/search/search-save/list/ui-list/saved-searches-list.ui-component.scss

```diff
@@ -0,0 +1,32 @@
+.aca-saved-searches-ui-list {
+  overflow-y: auto;
+
+  .adf-datatable-list {
+    border: 0;
+
+    .adf-datatable-header {
+      padding: 0 20px;
+
+      .adf-datatable-row {
+        margin-right: 15px;
+        padding-right: 0;
+        border-bottom: 1px solid var(--adf-theme-foreground-text-color-007);
```

Comment:
Hmm it's border but color's variable says 'text-color'. Is there any alternative color or can we introduce new variable/rename this?

Reply:
I saw that it was added in ADF by HX team as a generic color so that it can match and be used in any app so I think this naming convention was intended is it okay for you to keep it?

Reply:
ok so let's keep it in that case @MichalKinas